### ISSUE TITLE
Can't build gem

### ISSUE THREAD

**Issue by acrogenesis** [thoughtbot/administrate-field-image#1]
when running `gem build administrate-field-image.gemspec` I get this error

```
Invalid gemspec in [administrate-field-image.gemspec]: cannot load such file -- administrate/field/base
ERROR:  Error loading gemspec. Aborting.
```

**Comment by wicky-andrian** [comment]
same isue

**Comment by michalvalasek** [comment]
Me too. Even tho the gem is working when I install it directly from github via Bundler.

**Comment by balbinas** [comment]
Me too, any update?

**Comment by nickcharlton** [comment]
This was caused by a circular dependency and we've fixed that in #6. It'll be released shortly!
